platforms/nuttx/src/bootloader/microchip/mpfs/main.c

```diff
@@ -641,9 +640,9 @@ arch_do_jump(const uint32_t *app_base)
 #endif
 
 	/* Linux on harts 3,4 */
-	if (sbi_loaded && u_boot_loaded) {
-		_alert("Jump to U-boot 0x%lx\n", CONFIG_MPFS_HART4_ENTRYPOINT);
+	if (u_boot_loaded) {
 #if CONFIG_MPFS_HART3_ENTRYPOINT != 0xFFFFFFFFFFFFFFFF
+		_alert("Jump to U-boot 0x%lx\n", CONFIG_MPFS_HART4_ENTRYPOINT);
```

Comment:
misleading entrypoint in alert().. should be HART3 ?

Reply:
Thanks fixed. also updated the submodules to point to updated master/main branches